com/win32com/server/dispatcher.py

```diff
@@ -233,6 +233,13 @@ def _trace_(self, *args):
         win32api.OutputDebugString(str(args[-1]) + "\n")
 
 
+DispatcherWin32dbg_deprecation_message = """\
+The DispatcherWin32dbg dispatcher is deprecated!
+Please let me know if this is a problem.
```

Comment:
while we are touching this, maybe change this line to, eg, "please open an issue at https://github.com/mhammond/pywin32/ is this is a problem"?